tests/test_nicenumber.py

```diff
@@ -50,7 +51,25 @@ def test_to_human():
         else:
             assert result == expected_result
 
+def test_to_pandas():
+    """Test to_pandas function"""
+    f = nn.to_pandas
+    test_df = pd.DataFrame(np.array([['1_000', '1_000_000'], ['1_000_000_000', '1_000_000_000_000']]), columns=['A', 'B'])
+    
+    # test 'df' TypeError raised with wrong type
+    raises(TypeError, f, df=[1,2,3]).match('pd.DataFrame')
+
+    # test 'col_names' TypeError raised wth wrong input type
+    raises(TypeError, f, df=test_df, col_names=1).match('str or list')
+
+    # test 'col_names' ValueError raised wth wrong input values
+    raises(ValueError, f, df=test_df, col_names=['X']).match('not present')
+
+    # test 'transform_type' ValueError raised wth wrong input values
+    raises(ValueError, f, df=test_df, transform_type='wrong').match('invalid')
 
+    # test shape of dataframes is equal
+    assert nn.to_pandas(test_df).shape == test_df.shape    
 
```

Comment:
It would be good to have a test where you convert a df from numeric -> human -> back to numeric and confirm its the same as original

Reply:
added in latest commit

Reply:
In general I'd avoid nested function calls like that, makes it hard to read whats going on. should work fine though for a test.